### PR TITLE
Update reader registration design

### PR DESCRIPTION
**Pull request by thomasguillot**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR tweaks the design a little bit. Mainly the typography/vertical rhythm to follow an 8px baseline. There's probably more to tweak but this is a first pass 🙂

I also moved the text for the magic link sign up, hopefully it'll be sufficient to fix https://app.asana.com/0/1202719992789620/1202723823068199/f

### How to test the changes in this Pull Request:

1. Test the front-end
2. Switch to this branch
3. Re-test

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: includes/class-reader-activation.php [hunk 1]
```diff
@@ -597,7 +615,10 @@ public static function render_auth_form( $is_inline = false ) {
 							<?php _e( "We've recently sent you an authentication link. Please, check your inbox!", 'newspack' ); ?>
 						</p>
 						<p data-action="pwd">
-							<?php _e( 'Sign in below to verify your identity.', 'newspack' ); ?>
+							<?php \esc_html_e( 'Sign in below to verify your identity.', 'newspack' ); ?>
+						</p>
+						<p data-action="link">
+							<?php \esc_html_e( 'Get a link sent to your email address to sign in instantly without your password.', 'newspack' ); ?>
```
Comment (dkoo) [inline includes/class-reader-activation.php:621]: "your" in this context implies that the user has a password, but this isn't always the case.

```suggestion
							<?php \esc_html_e( 'Get a link sent to your email address to sign in instantly without a password.', 'newspack' ); ?>
```

File: assets/reader-activation/auth.scss [hunk 2]
```diff
@@ -140,51 +173,70 @@
...
+
+				> *:last-child {
+					margin-bottom: 0 !important;
+				}
 			}
 			input[type='email'],
 			input[type='password'] {
 				width: 100%;
 			}
-			[type='submit'] {
```
Comment (dkoo) [inline assets/reader-activation/auth.scss:184]: Was this intentionally removed? This makes the submit button take on the theme's secondary color, which by default is quite a bit less prominent than the primary.

With primary color (`master`):
Screen Shot 2022-08-08 at 9 22 49 PM

With secondary color (this PR):
Screen Shot 2022-08-08 at 9 23 10 PM

File: assets/reader-activation/auth.scss [hunk 3]
```diff
@@ -219,41 +269,48 @@
...
+				outline: 1.5px solid #111;
+				outline-offset: 2px;
 			}
 		}
 
 		&__terms-text {
-			font-size: 0.8em;
-			margin: 0.5rem 0;
-			color: #757575;
+			font-size: 0.625em;
```
Comment (dkoo) [inline assets/reader-activation/auth.scss:311]: This text is now minuscule! I feel like it's a bit _too_ small at 10px.

**Review by dkoo — CHANGES_REQUESTED** [review]
Definitely love the new placement of the magic link explanation text—it feels more consistent with the placement of the explanation in the password form and doesn't get lost as easily.

Other than that, I have some questions/suggestions about the other changes.

File: assets/reader-activation/auth.scss (hunk 2 above)
Comment (thomasguillot) [reply assets/reader-activation/auth.scss:184]: Yes, it's intentional. I wanted to match the theme's styles.
Also for now, we said we would use custom CSS to apply a red colour #DD3333 to these buttons.
That being said, I'd like to introduce a setting somewhere (Site Design? Reader Activation?) to control the background/text colour for them.

File: assets/reader-activation/auth.scss (hunk 3 above)
Comment (thomasguillot) [reply assets/reader-activation/auth.scss]: My bad, I wanted 11px. Fixed here: https://github.com/Automattic/newspack-plugin/pull/1870/commits/ec3611c059f10a0fb9bafcf2dc428b1cfca086dd

Comment (dkoo) [reply assets/reader-activation/auth.scss]: It's still pretty small—if it were up to me I'd make it 13px to match the size of the other "supplemental" text like "Lost your password?" But that's totally subjective, so I won't block the PR for this.

**Review by dkoo — APPROVED** [review]

**Pull request merged**